### PR TITLE
Fix BrowserAuthorizationLogger initialization

### PR DESCRIPTION
**Pull request by evelynpreslar-bentley**
Add explicit check for BrowserAuthorizationLogger initialization, instead of inferring the initialization state from the oidc-client library

### CODE REVIEW

File: clients/frontend-authorization/src/oidc/browser/BrowserAuthorizationLogger.ts [hunk 1]
```diff
@@ -57,7 +59,7 @@ export class BrowserAuthorizationLogger implements IOidcClientLogger {
   /** Initializes forwarding of OidcClient logs to the Bentley Logger */
   public static initializeLogger() {
     const logLevel = BrowserAuthorizationLogger.getLogLevel(FrontendAuthorizationClientLoggerCategory.Authorization);
-    if (!OidcClientLog.logger) {
+    if (!BrowserAuthorizationLogger.initialized) {
       OidcClientLog.logger = new BrowserAuthorizationLogger();
```
Comment (aruniverse) [inline clients/frontend-authorization/src/oidc/browser/BrowserAuthorizationLogger.ts:63]: should you also set initialized to true afterwards?

**Comment by aruniverse** [comment]
fyi @calebmshafer @skirby1996 we'll want this one backported as well

**Review by aruniverse — APPROVED** [review]

**Review by calebmshafer — APPROVED** [review]

**Comment by calebmshafer** [comment]
@Mergifyio update

**Comment by calebmshafer** [comment]
@Mergifyio update

**Pull request merged**